APIs_USB/USBHID/main.cpp

```diff
@@ -4,7 +4,7 @@
  */
 #include <stdio.h>
 #include "mbed.h"
-#include "drivers/USBHID.h"
+#include "USBHID.h"

```

Comment:
```suggestion
#include "usb/USBHID.h"
```

Reply:
@hugueskamba this change was necessitated because of the file being moved into a new path, is it really wise to revert back to an absolute path which may change in the future?

Reply:
This is not an absolute path, it is a relative path. We are using relative paths to give a namespace in case there are more than one header file with the same names.

Reply:
It shows which module the header file belongs to.
If relative paths were used at the beginning it would have been self-describing without requiring every module to prefix their name to header files (`usb/hid.h`, much nicer...)